docker-compose.yml

```diff
@@ -70,14 +72,25 @@ services:
       - POSTGRES_USER
       - POSTGRES_PASSWORD
       - POSTGRES_DB
+    ports:
+      - "5432:5432"
```

Comment:
I don't think we need to export this port on the host (e.g. we're not accessing postgres via `localhost` port 5432).